app/http.php

```diff
@@ -1015,6 +1016,24 @@ function (string $runtimeId, ?string $payload, string $path, string $method, arr
                 ];
             };
 
+            $pingStart = \microtime(true);
+            while (true) {
```

Comment:
How do we detect a cold start?

Reply:
Needs to be added, and only do this on cold-start.
With that in mind, I know there is loop for when 2 concurrent cold-start requests arrive. That loop also needs to become infinite, and respect `$timeout`

Reply:
Made preparation loop infinite, and it respects $timeout

Reply:
Added extra check to only do TCP check for the first execution